content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "CompilED for Inclusion"
+date: 2019-04-11
+type: "post"
+authors: ["mustapha"]
+tags: ["accessibility","inclusive design"]
+lede: "I restructured and redesigned the CTL DevTeam’s website CompilED to meet
+the web accessibility standards, and to improve its overall user experience
+inclusively on all devices."
+shortlede: "This site’s latest iteration has improved design and WCAG 2.0
+compliance."
+poster: "poster-compiled-v3.jpg"
+socmediaimg: ""
+hiliteimg: "poster-compiled-v3.jpg"
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: "dragon"
+---
+
+In March, I restructured and redesigned the CTL DevTeam’s website _CompilED_ to
+meet the web accessibility standards, and to improve its overall user
+experience inclusively on all devices.
+
+This latest iteration of _CompilED_ follows the
+[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/)
+level AA, and it is much more technically and usably accessible than the
+previous versions. It is keyboard accessible with improved focus styling. The
+site is also seamlessly adaptive to multi-size screen devices—smartphones,
+tablets, and desktops. The comprehensive design has also improved the
+readability of the site’s content.
+
+In my work at the CTL, I consider accessibility as an integral part of my own
+development process. Lately, I’ve been mulling over an effective, broader
+integration of accessibility thinking, process, and policy across different
+groups in the whole department, as part of our commitment to accessibility,
+universal design, and inclusion in higher education.
+
+[_CompilED_ redesign in 2016](/articles/rebuilding-compiled/) afforded some
+accessibility since my grasp of WCAG standards was limited at the time. Three
+years later with more experience and grounded understanding, I redesigned
+_CompilED_ as an exercise to develop a website that conforms very well to the
```

Comment:
I might change 'conforms very well' to 'strongly conforms', or swaps adverb and verb: 'very well conforms'.

Reply:
I'm strongly holding on to my non-committal commitment.